### ISSUE TITLE
[Feature Request]: Add the ability to remove/edit treatments

### ISSUE THREAD

**Issue by joyrida08** [nightscout/cgm-remote-monitor#637]
##

**Comment by jasoncalabrese** [comment]
We've been waiting on getting auth in place before adding editing, hopefully soon.  Editing in mongo is getting old...

**Comment by joyrida08** [comment]
Ahh OK, makes perfect sense. I only have a couple entries but I jacked one
up already and realized I'm probably going to edit often lol.

I wish I knew what the hell I was doing in JavaScript so I could help y'all
out but my knowledge only gets me far enough to be able to get a general
idea of what's going on lol.

**Comment by jasoncalabrese** [comment]
You can edit and delete from the mongolab site

![screen shot 2015-06-18 at 7 56 42 pm](https://cloud.githubusercontent.com/assets/751143/8246496/5a133d28-15f4-11e5-81ec-7bf9baa533dc.png)

![screen shot 2015-06-18 at 7 57 00 pm](https://cloud.githubusercontent.com/assets/751143/8246498/5e52f176-15f4-11e5-91ac-f575fe132e31.png)

![screen shot 2015-06-18 at 7 57 25 pm](https://cloud.githubusercontent.com/assets/751143/8246499/62419bf2-15f4-11e5-9c1c-f4dc1729c832.png)

**Comment by joyrida08** [comment]
Yeah, I figured that out. Took a little bit of reading through the source
code to realize it though. I don't know why I didn't think to look there in
the first place knowing that's where all the other data is being stored lol.

**Comment by whoiswes** [comment]
This may need it's own issue opened, but as long as we're talking treatments, I would also vote for the inclusion of a date field in addition to time for entering data.  It's not always possible to enter things on the same day, so it would be nice to be able to specify the date as well as the time.

We're only 2 months into our Type 1 Odessey with our 3 year old son, and I haven't had time to peruse much code (and I'm not much of a dev anyway, I dabble in Python, PHP, and Powershell mainly and am a complete newb when it comes to git and versioning) but if I come up with something workable I'll figure out how to share.

Feel free to delete/move this comment if needed.

**Comment by jasoncalabrese** [comment]
@whoiswes think that is a new issue, but it is something I've wanted to get in, and we can do it now before auth is in place

**Comment by dbeasy** [comment]
I was reading through the issues list, can we close this one, it's basically fixed with the link to the Reports (then you have to wait for the green tabs, then pick "Treatments" tab, then click on "newest on top" (I fixed that default in my branch) and then click on the "show" button). The other old issue, about date AND TIME, was fixed long ago -- I do remember a lot of late nights, early on, making sure I had the careportal documentation submitted for the patient before midnight.

Or should this be left open until there is a button like "Reports" but goes directly into the "Treatments" tab and shows the most 2 days recent Careportal updates by default?

**Comment by Kdisimone** [comment]
This issue could be closed.  The treatments can be edited on the Reports section, Treatments tab
